.github/workflows/ci-all-in-one-build.yml

```diff
@@ -6,9 +6,14 @@ on:
   pull_request:
     branches: [ master ]
 
-jobs:
+jobs:   
   all-in-one:
     runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
```

Comment:
Yes, we still need to publish the ports, otherwise we couldn't access the local registry service (localhost:5000) to hold multi-arch `baseimg`, and push and pull multi-arch `all-in-one` image while running local integration test.